server.js

```diff
@@ -0,0 +1,21 @@
+Meteor.methods({
+  'phoneformat.getCountryForIp': function () {
```

Comment:
http://stackoverflow.com/a/23027337/230462 <-- gets the client IP address. Also please add a jsdoc comment for the method